test/functional/services/test_subjects.ts

```diff
@@ -119,12 +117,10 @@ export function TestSubjectsProvider({ getService }: FtrProviderContext) {
     }
 
     public async doubleClick(selector: string, timeout: number = FIND_TIME): Promise<void> {
-      return await retry.try(async () => {
-        log.debug(`TestSubjects.doubleClick(${selector})`);
-        const element = await this.find(selector, timeout);
-        await element.moveMouseTo();
-        await element.doubleClick();
-      });
+      log.debug(`TestSubjects.doubleClick(${selector})`);
+      const element = await this.find(selector, timeout);
+      await element.moveMouseTo();
+      await element.doubleClick();
     }
```

Comment:
WebElementWrapper has `retryCall` with 3 attempts to finish the action:
```
  public async doubleClick() {
    await this.retryCall(async function clickMouseButton(wrapper) {
      await wrapper.scrollIntoViewIfNecessary();
      await wrapper
        .getActions()
        .doubleClick(wrapper._webElement)
        .perform();
    });
  }
```

I don't think we need to use retry service here